src/test/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/client/ChipsRestClientTest.java

```diff
@@ -33,7 +33,7 @@
     private static final String CHIPS_REST_HOST = "hostname/";
     private static final String CHIPS_REST_ENDPOINT = "test-endpoint";
     private static final String DATA = "data";
-    private static final URI FULL_EXPANDED_CHIPS_REST_URL = new UriTemplate(CHIPS_REST_HOST+CHIPS_REST_ENDPOINT).expand();
+    private static final URI FULL_EXPANDED_CHIPS_REST_URL = new UriTemplate(CHIPS_REST_HOST + CHIPS_REST_ENDPOINT).expand();
```

Comment:
Could we get a String.format here?

Reply:
Changing the code isn't really within the scope of this pr, just reformatting so the code better fits any coding standards.

As for adding a String.format i'd argue that it makes the code more difficult to read and doesn't offer any performance improvements. So I would argue against this being a string format

Reply:
While String.format is better in a lot of cases, in this case it is proabably not; don't want to over promote this so will approve.